### ISSUE TITLE
Sqlite3 - Generate more type of statements during initialisation 

### ISSUE THREAD

**Issue by JonathanFoo0523** [sqlancer/sqlancer#944]
At initialisation of sqlite3 db, sqlancer insert random number of different sql statement specified by 
```
private static int mapActions(SQLite3GlobalState globalState, Action a) {
        int nrPerformed = 0;
        Randomly r = globalState.getRandomly();
        switch (a) {
        case CREATE_VIEW:
            nrPerformed = r.getInteger(0, 2);
            break;
        case DELETE:
        case DROP_VIEW:
        case DROP_INDEX:
            nrPerformed = r.getInteger(0, 0);
            break;
        case ALTER:
            nrPerformed = 0;
            break;
        case EXPLAIN:
        case CREATE_TRIGGER:
        case DROP_TABLE:
            nrPerformed = r.getInteger(0, 0);
            break;
        case VACUUM:
        case CHECK_RTREE_TABLE:
            nrPerformed = r.getInteger(0, 3);
            break;
        case INSERT:
            nrPerformed = r.getInteger(0, globalState.getOptions().getMaxNumberInserts());
            break;
        case MANIPULATE_STAT_TABLE:
            nrPerformed = r.getInteger(0, 5);
            break;
        case CREATE_INDEX:
            nrPerformed = r.getInteger(0, 5);
            break;
        case VIRTUAL_TABLE_ACTION:
        case UPDATE:
            nrPerformed = r.getInteger(0, 30);
            break;
        case PRAGMA:
            nrPerformed = r.getInteger(0, 20);
            break;
        case CREATE_TABLE:
        case CREATE_VIRTUALTABLE:
        case CREATE_RTREETABLE:
            nrPerformed = 0;
            break;
        case TRANSACTION_START:
        case REINDEX:
        case ANALYZE:
        case ROLLBACK_TRANSACTION:
        case COMMIT:
        default:
            nrPerformed = r.getInteger(1, 10);
            break;
        }
        return nrPerformed;
    }
```

Is there any reason we don't generate statements such as `CREATE_VIRTUALTABLE` and `ALTER` during initialisation? By changing `ALTER` to be generated  `r.getInteger(1, 10)` times, and add expected error for `check constraints`, we could generate more diverse statements.

**Comment by mrigger** [comment]
Sorry, I overlooked this issue. I responded to you in Slack in the meanwhile.

For `CREATE_VIRTUALTABLE`, this is deliberate as we create tables as a first step, and doing so makes it easier to control the overall number of tables of different types.

For `ALTER`, It is likely that I disabled it at some point since this statement no longer triggered bugs, and made it more difficult to reduce the test cases. I think we could enable them again (but perhaps limited the maximum number to 2).
